pipeline_api/src/main/java/com/splicemachine/pipeline/client/WriteResult.java

```diff
@@ -85,7 +85,13 @@ public boolean shouldCancel() {
         return code.shouldCancel();
     }
     public boolean canRetry() {
```

Comment:
How do we control the maximum number of retries? What happens if max retries are exceeded.

Reply:
It will be retired infinitely